.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -0,0 +1,114 @@
+name: 🐛 Bug Report
+description: Submit a bug report to help us improve
+labels: [bug, triage]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Please help us help you!
+
+        Before filing your issue, ask yourself:
+
+        - Is this clearly a Pact Python bug?
+        - Do I have basic ideas about where it goes wrong?
+        - Could it be because of my own mistakes?
+
+        **The GitHub issue tracker is not a support forum**. If you are not sure whether it could be your mistakes, ask on [Slack](https://slack.pact.io).
+
+        Make the bug obvious. Ideally, we should be able to understand it without running any code.
+
+        Bugs are fixed faster if you include:
+        - A reproduction repository to replicate the issue
+        - Pact files
+        - Logs
```

Comment:
110% - we might want to consider a _repro_ repo they can fork, or how they can add their example say to your example suite, and have the ci system run it for a repro, that way it is contained within the maintainers domain (of pact-python) and if the examples are useful they can remain

Reply:
Completely agree, but I think this can be addressed later (want me to create an issue to track this?)

Reply:
Yeah one for addressing later, and ticket to track would be good 👍🏾 decreases the cognitive load we carry

Reply:
Done, created a ticket 😁